bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/chlorinator.xml

```diff
@@ -77,31 +76,27 @@
 		<label>Chlorinator Alert</label>
 		<description>Chlorinator Alert</description>
 		<state readOnly="true">
-			<options>
-				<option value="0">None</option>
-				<option value="16">Low T-Cell Temperature</option>
-			</options>
 		</state>
 	</channel-type>
 
 	<channel-type id="avgSaltLevel">
 		<item-type>Number:Dimensionless</item-type>
 		<label>Average Salt Level</label>
 		<description>Average Salt Level</description>
-		<state pattern="%d %unit%" readOnly="true"/>
+		<state pattern="%d ppm" readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="instantSaltLevel">
 		<item-type>Number:Dimensionless</item-type>
 		<label>Instant Salt Level</label>
 		<description>Instant Salt Level</description>
-		<state pattern="%d %unit%" readOnly="true"/>
+		<state pattern="%d ppm" readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="status">
-		<item-type>Number</item-type>
+		<item-type>String</item-type>
 		<label>Status</label>
 		<description>Status</description>
-		<state pattern="%d" readOnly="true"/>
+		<state readOnly="true"/>
```

Comment:
You could keep a pattern but with %s.
That is always good to define a default pattern.